src/components/Form/Form.js

```diff
@@ -3,9 +3,9 @@ import React from 'react';
 // Start Uppercase
 // export default should be added
 export default function From() {
-	return (
-		<>
-			<span>My Form</span>
-		</>
-	);
-};
+    return ( 
+        <>
+        <span > My Form </span> 
```

Comment:
Hey, @olfasaidi here you could add an indentation, which would be a nice practice.
example:
```
<parent>
        <child/>
</parent>
``` #1

Reply:
@fToledo29 i fixed the issue .